test/src/main/java/org/apache/accumulo/test/fate/FateExecutionOrderIT_SimpleSuite.java

```diff
@@ -55,21 +55,19 @@
 import org.apache.accumulo.core.fate.FateId;
 import org.apache.accumulo.core.fate.FateStore;
 import org.apache.accumulo.core.fate.Repo;
-import org.apache.accumulo.harness.AccumuloITBase;
 import org.apache.accumulo.harness.SharedMiniClusterBase;
 import org.apache.accumulo.server.ServerContext;
 import org.apache.hadoop.io.Text;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import com.google.common.collect.Iterators;
 
-@Tag(AccumuloITBase.SIMPLE_MINI_CLUSTER_SUITE)
-public abstract class FateExecutionOrderIT extends SharedMiniClusterBase
-    implements FateTestRunner<FateExecutionOrderIT.FeoTestEnv> {
+public abstract class FateExecutionOrderIT_SimpleSuite extends SharedMiniClusterBase
+    implements FateTestRunner<FateExecutionOrderIT_SimpleSuite.FeoTestEnv> {
+  // TODO KEVIN RATHBUN double check that the sub classes don't need to be simple suite
```

Comment:
Need to resolve/remove this TODO before merging.

Reply:
Left that to remind me to check that this test was actually added to the suite. Turns out it wasn't. Only the concrete classes should be marked "IT_SimpleSuite". I addressed this in 87e90333dde61ea619e82510ae9c544e71a10722 (only this one test was incorrect). I also added a note in the javadocs for the suite explaining that only concrete classes should be marked.